bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -133,6 +134,35 @@ trait UtilBijections {
     }
   }
 
+  /**
+    * Bijection between java completable futures and twitter futures.
+    */
+  implicit def twitter2CompletableFutureBijection[A]
+      : Bijection[TwitterFuture[A], CompletableFuture[A]] = {
+    new AbstractBijection[TwitterFuture[A], CompletableFuture[A]] {
+      override def apply(f: TwitterFuture[A]): CompletableFuture[A] = {
+        val cf = new CompletableFuture[A]()
+        f.respond {
+          case Return(value)    => cf complete value
```

Comment:
minor nit, can we avoid infix (which is going away in scala 3). `cf.complete(value)`

Reply:
No problem, do you want me to update the other places in this file where infix is being used? I mainly used it here due to its use elsewhere in the file.

Reply:
I understand.

No need to change the previous code. We can run a scalafix or scalafmt or something to get other use cases, I just don't want to add new ones. I don't want to complicate the PR with other changes, but also don't want to dig in deeper on the old pattern.